modules/ROOT/pages/sync-api-projects-design-center.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio 7.5 and later enables you to import a RAML or OAS API specification from Design Center into an API Project in Studio to edit it offline, and stage and push your changes as you would do with any version control system (VCS).
+Anypoint Studio enables you to import a RAML, OAS 2.0, or OAS 3.0 API specification from Design Center into an API Project in Studio to edit it offline, and stage and push your changes as you would do with any version control system (VCS).
```

Comment:
```suggestion
Anypoint Studio enables you to export a RAML, OAS 2.0, or OAS 3.0 API specification from Design Center to an API project in Studio so that you can edit it offline and then stage and push your changes as you would do with any version control system (VCS).
```